contracts/bitizens/AvatarItemOperator.sol

```diff
@@ -1,48 +1,132 @@
     itemService = AvatarItemService(_addr);
   }
 
-  function createAvatarItem( 
+  function getOwnedTokenIds() external view returns(uint256[] _tokenIds) {
+    return itemService.getOwnedTokenIds(msg.sender);
+  }
+
+  function getTokenInfo(uint256 _tokenId)
+    external 
+    view 
+    returns(address, address, bool, int16, uint256[4] _attr1, uint8[5] _attr2) {
+    return itemService.getTokenInfo(_tokenId);
+  }
+
+  function isBurned(uint256 _tokenId) external view returns (bool) {
+    return itemService.isBurned(_tokenId);
+  }
+
+  function getBurnedTokenCount() external view returns (uint256) {
+    return itemService.getBurnedTokenCount();
+  }
+  
+  function getBurnedTokenIdByIndex(uint256 _index) external view returns (uint256) {
+    return itemService.getBurnedTokenIdByIndex(_index);
+  }
+
+  function getSameItemTokenIds(uint256 _tokenId) external view returns(uint256[]) {
+    return itemService.getSameItemTokenIds(_tokenId);
+  }
+
+  function isSameItem(uint256 _tokenId1, uint256 _tokenId2) external view returns (bool) {
+    return itemService.isSameItem(_tokenId1,_tokenId2);
+  }
+
+  function getMineTrack(uint256 _tokenId) external view returns (uint256[]){
+    return itemService.getMineTrack(_tokenId);
+  }
+
+  function burnToken(uint256 _tokenId) external {
+    itemService.burnToken(msg.sender, _tokenId);
+    emit BurnToken(msg.sender, _tokenId);
+  }
+
+  function createToken( 
     address _owner,
-    uint32 _listNumber,
-    uint8 _setNumber,
-    bool _isBitizenItem, 
-    bool _isSet, 
-    uint8 _types,
-    uint8 _rarity,
-    uint8 _socket,
-    uint8 _gender,
-    uint8 _ext1,
-    uint8 _ext2
-    ) external onlyOperator returns(uint256 _tokenId)  {
-    _tokenId = itemService.createAvatarItem(_owner,_listNumber,_setNumber,_isBitizenItem,_isSet,_types,_rarity,_socket,_gender,_ext1,_ext2);
-    emit AvatarItemCreateSuccess(msg.sender, _tokenId);
-  }
-
-  function getOwnedTokenIds() external view returns(uint256[] _tokenIds){
-    _tokenIds = itemService.getOwnedTokenIds(msg.sender);
-  }
-
-  function getAvatarItemInfo(uint256 _tokenId)
+    address _founder,
+    address _creator,
+    bool _isBitizenItem,
+    int16 _probability,
+    uint256[4] _attr1,
+    uint8[5] _attr2) 
     external
-    view 
-    returns(
-    uint32 _listNumber,
-    uint16 _setNumber,
-    bool _isBitizenItem, 
-    bool _isSet,
-    uint8[6] _attr){
-    (_listNumber,_setNumber,_isBitizenItem,_isSet,_attr) = itemService.getAvatarItemInfo(_tokenId);
+    onlyOwner
+    returns(uint256 _tokenId) {
+    _tokenId = _mintToken(_owner, _founder, _creator, _isBitizenItem, _probability, _attr1, _attr2);
+    emit CreateToken(_owner, _tokenId);
+  }
+  
+  function batchCreateToken(
+    address _owner,
+    bool[] _isBitizenItems,  // base length
+    int16[] _probabilities,  // one time
+    address[] _addresses,    // two times 
+    uint256[] _attrs1,       // four times 
+    uint8[] _attrs2          // five times
```

Comment:
Readability: "five times" you meant "base length x 5"? you have this check below, you can add the comments below. The comments here should be what the fields are for or meant, not the validation rules.

Reply:
Thanks your good advice